paymentsheet/res/values/strings.xml

```diff
@@ -6,6 +6,10 @@
   <string name="stripe_bank_account_ending_in">Remove bank account ending in %s</string>
   <!-- Text for alert message when user needs to confirm payment in their banking app -->
   <string name="stripe_blik_confirm_payment">Confirm the payment in your bank\'s app within %s to complete the purchase.</string>
+  <!-- Used as the title for prompting the user if they want to close the sheet -->
+  <string name="stripe_confirm_close_form_body">Your payment information will not be saved.</string>
+  <!-- Used as the title for prompting the user if they want to close the sheet -->
+  <string name="stripe_confirm_close_form_title">Do you want to close this form?</string>
```

Comment:
This should be a bit more concise.

```suggestion
  <string name="stripe_confirm_close_form_title">Close this form?</string>
```

Reply:
This is what iOS is using, so I will keep it like this.